### DISCUSSION TITLE
有没有电脑是linux或者MAC的，wine 可以运行这个程序吗？

### DISCUSSION THREAD

**Discussion by saltyduckegg** [NateScarlet/auto-derby#254] [Q&A] [answered]
有没有大佬有相关经验，我不知道wine该如何来安装python依赖的库。

**Comment by haotingliulime** [comment, marked as answer]
事实上你只能win跑，因为你需要dmm来跑游戏。。。[python 3.8](https://www.python.org/downloads/release/python-380/)
